### ISSUE TITLE
Suggestion: check if NPC is male/female (perhaps from the wiki) and set random voice to that gender

### ISSUE THREAD

**Issue by fpartous** [Bullish53/TTS#5]
First of all, fantastic plugin! I had been looking for something to narrate the dialogues when questing for a long time, as it improves the immersion. However now when I chose random voice for dialogues it can use a male voice for a female NPC and vice-versa. Perhaps there can be a way to get a classifications of the NPCs genders and use the appropriate voice based on that? 
On top of that perhaps voice types can even depend on the continent on which the player is, for example a spanish voice type could be chosen for a southern continent like and a swedish voice type for a northern region.

**Comment by Bullish53** [comment]
Yes i planned to add this to NPC's too but the game only knows if players are male or female but not for NPC's so maybe in the future i can add some wiki scraper to get that data and add the voices accordingly
